libswirl/hw/pvr/ta_ctx.cpp

```diff
@@ -15,26 +15,28 @@ tad_context ta_tad;
 TA_context*  vd_ctx;
 rend_context vd_rc;
 
-#if ANDROID
-#include <errno.h>
-#include <malloc.h>
-
-int posix_memalign(void** memptr, size_t alignment, size_t size) {
-  if ((alignment & (alignment - 1)) != 0 || alignment == 0) {
-    return EINVAL;
-  }
-
-  if (alignment % sizeof(void*) != 0) {
-    return EINVAL;
-  }
-
-  *memptr = memalign(alignment, size);
-  if (*memptr == NULL) {
-    return errno;
-  }
-
-  return 0;
-}
+#ifdef __ANDROID__
```

Comment:
Since you're touching this, can you move it to the android-specific platform code?

Reply:
Into `libswirl/android/Android.cpp`?